tests/manage/pv_services/test_pvc_assign_pod_node.py

```diff
@@ -21,6 +22,25 @@ class TestPvcAssignPodNode(ManageTest):
     OCS-1257 - RBD: Assign nodeName to a POD using RWX PVC
     """
 
+    @bugzilla("2136852")
+    def verify_access_token_notin_odf_pod_logs(self):
```

Comment:
this test may fail on <4.12 cluster as the fix for this issue is present only in 4.12

Reply:
done